cmd/job-executor-service/main.go

```diff
@@ -186,8 +191,17 @@ func _main(args []string, env envConfig) int {
 		log.Fatalf("failed to create client, %v", err)
 	}
 
+	imageFilterList, err := buildImageAllowList(env.AllowedImageList)
+	if err != nil {
+		log.Fatalf("failed to generate the allowlist, %v", err)
+	}
+
+	processCloudEventFunc := func(ctx context.Context, event cloudevents.Event) error {
+		return processKeptnCloudEvent(ctx, event, imageFilterList)
+	}
+
 	log.Printf("Starting receiver")
-	log.Fatal(c.StartReceiver(ctx, processKeptnCloudEvent))
+	log.Fatal(c.StartReceiver(ctx, processCloudEventFunc))
```

Comment:
Why renaming the func?

Reply:
The function must be wrapped, because I added another parameter (the allow list) to it: 
https://github.com/keptn-contrib/job-executor-service/pull/213/files/7e55ed3bca3b2746a21f55a50f9667c5657a80f0#diff-2c166bc577041c1b2a355e339b9fe7e6d7e088b3e3acaaa32957118a29ac1652R199-R201